src/common/DeveloperConfig.ts

```diff
@@ -112,6 +114,9 @@ export type DeveloperConfig = {
     educationContentOverrides?: Record<string, EducationContent>;
   };
   onAppSessionStart?: (params: OnAppSessionStartParams) => Promise<void>;
+  modifySettingScreenMenuItems?: (
+    items: { id: string; title: string; action: () => void }[],
+  ) => { id: string; title: string; action: () => void }[];
```

Comment:
So you get the default SDK menu items passed to you, then return the ones you want shown.  Beautiful.

Reply:
Yeah exactly. It also has the added benefit of being able to change the default order in addition to injecting/removing default values.